src/main/java/com/avioconsulting/mule/vault/provider/api/VaultConfigurationPropertiesProvider.java

```diff
@@ -40,10 +41,12 @@ public class VaultConfigurationPropertiesProvider implements ConfigurationProper
 
     /**
      * Constructs a VaultConfigurationPropertiesProvider. Vault must not be null.
+     * If isLocalMode is true, it will use the fallback file which contain secretPath and secrets
+     * in case vault server connections are not available.
```

Comment:
We should add that it will completely disable connections to the Vault server when localMode is true also